### DISCUSSION TITLE
Online Hosting of Shields using IIS

### DISCUSSION THREAD

**Discussion by Ruairi-azyra** [badges/shields#6990] [Q&A] [answered]
Is it possible to web host a shilds application using Internet Information Services.

**Comment by calebcartwright** [comment, marked as answer]
Presumably possible, though I don't know why you'd want to use IIS for our app and would advise against it. There's myriad blog posts on the internet about running Node.js apps with IIS you can reference, but we're not going to do any troubleshooting nor provide any guidance on doing so.

You can find our docs on self-hosting in https://github.com/badges/shields/blob/master/doc/self-hosting.md which includes options you'll likely find much simpler unless you are absolutely coupled to the Windows ecosystem as a host env.
